.gitignore

```diff
@@ -1,29 +1,31 @@
-*.Rcheck
-*.html
-*.rprof
-*.sas.txt
-*~
-.DS_Store
-.RData
-.Rhistory
-.Rproj.user
-.Ruserdata
-.httr-oauth
-.project
-.settings/**
-/.project
-Meta
-coverage.*
-devel/*
-doc
-docs
-inst/outputs/*
-logs
-packrat/lib*/
-temp
-temp_w
-templates/
-tmp.*
-vignettes/*.R
-vignettes/*.html
-vignettes/*.md
+*.Rcheck
```

Comment:
Do we have a standardized .gitignore? Should we also propagate this throughout the other repos?

Reply:
removed, it was automatically added by pkgdown?